git-coco

```diff
@@ -164,6 +194,61 @@ pr() {
     echo "https://console.aws.amazon.com/codesuite/codecommit/repositories/${repo}/pull-requests/${id}/changes"
 }
 
+prlist() {
+    local repo list_of_prs pull_info
+    local dst_branch src_branch is_merge pull_id author title
+
+    repo="$1"
+
+    # Handle default case
+    if [ -z "$repo" ]; then
+        local urls=$(git remote -v | awk -F ' ' '{ print $2; }')
+
+        for url in $urls; do
+            repo="$(url_parser "$url")"
+
+            # Break out of loop if valid repo is found
+            if [[ ! -z "$repo" ]]; then
+                break
+            fi
+        done
+
+        if [ -z "$repo" ]; then
+            fail "No repository provided. Current directory does not have a valid codecommit repository."
+        fi
+    fi
+
+    local header="%-10s %-60s %-8s %-100s\n"
+    local format="%-10s %-60s %-8s %s <- %s (%s)\n"
+    local pr_query="pullRequest.{PR: pullRequestId, Title: title, Merge: pullRequestTargets[0].mergeMetadata.isMerged, DstBranch: pullRequestTargets[0].destinationReference, SrcBranch: pullRequestTargets[0].sourceReference, Author: authorArn}"
```

Comment:
I think the "isMerged" field isn't useful here since the PR is open (`--pull-request-status OPEN` is specified below) -- what would be useful (but require additional API calls) is if it's mergable, but maybe it's better to just reduce the information shown

Reply:
Good point, I'll leave it as is with just pulling out merge status. If anything pops up that seems useful to include in that table I'll probably add it in the future.